src/swarm-tui/Swarm/TUI/Controller.hs

```diff
@@ -1458,6 +1458,7 @@ handleInventoryListEvent ev = do
   -- However, this does not work since we want to skip redrawing in the no-list case!
 
   mList <- preuse $ uiState . uiGameplay . uiInventory . uiInventoryList . _Just . _2
+  resetViewport infoScroll
```

Comment:
This looks to me like it will still reset the viewport on *every* keypress in the inventory list, which would include *e.g.* hitting `m` to `make` the currently focused item or hitting some random key that has no effect.  We should only reset the viewport when actually moving to a different item.

Reply:
You were right. The reason I failed to catch this bug in my testing was because it didn't occur when hitting defined key press events. Just the random ones which I never tested earlier 🤦